model/BUILD.bazel

```diff
@@ -98,6 +98,7 @@ proto_library(
 
 java_lite_proto_library(
     name = "subtitled_html_java_proto_lite",
+    visibility = ["//visibility:public"],
```

Comment:
Is there a way we can make this a bit more restricted?

Reply:
I think we could actually remove this line, since the proto is getting indirectly exported through the exploration proto.

Reply:
I'll remove and build. If it brings errors I'll peruse the visibility folder to select the appropriate visibility for it.